ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -54,23 +55,8 @@ public enum CocoaPodUtils {
     }
   }
 
-  /// Executes the `pod cache list` command to get the Pods curerntly cached on your machine.
-  ///
-  /// - Parameter dir: The directory containing all installed pods.
-  /// - Returns: A dictionary keyed by the pod name, then by version number.
-  public static func listPodCache(inDir dir: URL) -> [String: [String: PodInfo]] {
-    let result = Shell.executeCommandFromScript("pod cache list", outputToConsole: false)
-    switch result {
-    case let .error(code):
-      fatalError("Could not list the pod cache in \(dir), the command exited with \(code). Try " +
-        "running in Terminal to see what's wrong.")
-    case let .success(output):
-      return parsePodsCache(output: output.components(separatedBy: "\n"))
-    }
-  }
-
   /// Gets metadata from installed Pods. Reads the `Podfile.lock` file and parses it.
-  public static func installedPodsInfo(inProjectDir projectDir: URL) -> [PodInfo] {
+  public static func installedPodsInfo(inProjectDir projectDir: URL) -> [String: PodInfo] {
```

Comment:
Can you mention what the key is in the documentation?

Reply:
Added a func doc block.